recipes/metpy/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
MetPy uses setuptools, right?  You can add ` --single-version-externally-managed --record record.txt` to that command then. (And the one below too.)